### ISSUE TITLE
Multiple gpus

### ISSUE THREAD

**Issue by zzzdeb** [grmat/amdgpu-fancontrol#28]
Nice script. Is it possible to enable multigpu control. For now I uses 2 binary.

**Comment by cfw34683** [comment]
yes i am also looking for this as well. i am running 4 terminals for all cards at the moment.. this is a great script

**Comment by denialo** [comment]
Hello,

how can i adjust this for multiple graphic cards please

# hwmon paths, hardcoded for one amdgpu card, adjust as needed
FILE_PWM=$(echo /sys/class/drm/card0/device/hwmon/hwmon?/pwm1)
FILE_FANMODE=$(echo /sys/class/drm/card0/device/hwmon/hwmon?/pwm1_enable)
FILE_TEMP=$(echo /sys/class/drm/card0/device/hwmon/hwmon?/temp1_input)

**Comment by grmat** [comment]
There is currently no support for multiple GPUs in this script. You can work around it by running multiple instances of the script with different configs.

**Comment by denialo** [comment]
is it possible just to run the script once and set the gpu fan speed to fixed 80% without keeping the script running in a loop ?

**Comment by grmat** [comment]
The whole purpose of this script is to run continuously. If you want to set a fixed speed, you'd just
`echo 204 > /sys/class/drm/card0/device/hwmon/hwmon0/pwm1`

**Comment by denialo** [comment]
Thanks, but that does not work for me, could you please advise


root@bana:/home/banik/amdgpu-fancontrol-master# echo 204 > /sys/class/drm/card2/device/hwmon/hwmon2/pwm1
bash: echo: write error: Invalid argument
root@bana:/home/banik/amdgpu-fancontrol-master# echo 204>/sys/class/drm/card2/device/hwmon/hwmon2/pwm1

root@bana:/home/banik/amdgpu-fancontrol-master# echo -n 204>/sys/class/drm/card2/device/hwmon/hwmon2/pwm1
root@bana:/home/banik/amdgpu-fancontrol-master# cat /sys/class/drm/card2/device/hwmon/hwmon2/pwm1
58
root@bana:/home/banik/amdgpu-fancontrol-master#

**Comment by grmat** [comment]
You have to set the PWM control from auto to manual first by writing `1` to `pwm1_enable`

**Comment by vypinac** [comment]
It works, thank you Sir.
